packages/redux/src/checkout/__tests__/reducer.test.ts

```diff
@@ -1326,6 +1326,80 @@ describe('checkout reducer', () => {
         ).toEqual(result);
       });
 
+      it('should reset address state code and name if they are not filled', () => {
+        const result = {
+          checkoutOrders: {
+            1: {
+              checkout: 123,
+              tags: ['NEW_GIFT'],
+              shippingAddress: {
+                state: {},
+                name: '',
+              },
+              billingAddress: {
+                state: {},
+              },
+            },
+          },
+        };
+        const state = {
+          checkoutOrders: {
+            1: {
+              checkout: 123,
+              tags: ['GIFT'],
+              shippingAddress: {
+                state: {
+                  code: 'code',
+                  name: 'name',
+                },
+              },
+              billingAddress: {
+                state: {},
+              },
+            },
+          },
+        };
+
+        expect(
+          // @ts-expect-error
```

Comment:
Why was this `ts-expect-error` necessary? If there is a valid reason, add the reason in front of it as it let others know why was it necessary without removing the comment.